src/classes/HH_Container_LCTRL.cls

```diff
@@ -142,12 +170,24 @@ public class HH_Container_LCTRL {
                 //we use dynamic soql so we can include all contact fields, since custom naming may refer to any field.
                 list<string> listStrField = getContactFields();
                 string strSoql = 'SELECT ';
-                string strComma = '';
+                string strComma = '';                
+                set<String> setCheckUpdate = setUpdateableContactFields();
+                
                 for (string strF : listStrField) {
+
+                    // check field level security
+                    DescribeFieldResult dfr = UTIL_Describe.getFieldDescribe('Contact', strF);
```

Comment:
I originally did code parsing for object.field and then realized that we were not needing to support lookup field references in any of the use cases in manage household ui, so I removed it to keep the code simpler.